src/config/mainnet.ts

```diff
@@ -1,19 +1,88 @@
-import { NetworkConfig } from './types';
+import { ZeroAddress } from 'ethers';
+import { Decimal } from '@tempusfinance/decimal';
+import PositionManagerABI from '../abi/PositionManager.json';
+import PositionManagerStEthABI from '../abi/PositionManagerStETH.json';
+import { NetworkConfig, TokenTickerToTokenConfigMap } from './types';
+import { getWstEthToStEthRate } from './rates';
+
+const tokenTickerToTokenConfigMap: TokenTickerToTokenConfigMap = {
+  WETH: {
+    address: '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2',
+    positionManager: '0x5f59b322eb3e16a0c78846195af1f588b77403fc',
+    positionManagerABI: JSON.stringify(PositionManagerABI),
+    positionMangerManageFunc: 'managePosition',
+    ticker: 'WETH',
+    supportsPermit: false,
+    priceFeedTicker: 'WETH',
+    hardcodedPrice: null,
+    underlyingTokenTicker: 'WETH',
+    underlyingCollateralRate: Decimal.ONE,
+  },
+  ETH: {
+    address: ZeroAddress,
+    positionManager: '', // waiting for delegate deployment
+    positionManagerABI: '', // add ABI once contract is deployed and verified
+    positionMangerManageFunc: '', // add once contract is deployed and function name is confirmed
+    ticker: 'ETH',
+    supportsPermit: false,
+    priceFeedTicker: null,
+    hardcodedPrice: null,
+    underlyingTokenTicker: 'WETH',
+    underlyingCollateralRate: Decimal.ONE,
+  },
+  wstETH: {
+    address: '0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0',
+    positionManager: '0x5f59b322eb3e16a0c78846195af1f588b77403fc',
+    positionManagerABI: JSON.stringify(PositionManagerABI),
+    positionMangerManageFunc: 'managePosition',
```

Comment:
Not a fan of this TBH. I'd better solve it as a separate class for each token type, rather than stringifying ABI. Looks like prone to errors.